deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -2,14 +2,39 @@
 # docker image tag if not overridden with imageTag
 agentVersion: 5.1.3
 
-# The access token for SignalFx. (REQUIRED if signalFxAccessTokenSecretName not set)
+# The access token for SignalFx.
 signalFxAccessToken: ""
 
 # Use an existing secret for the SFX token (if set does not create a secret)
 # See templates/secrets.yaml for the format of the secret to use
-# If set, signalFxAccessToken is not required
 signalFxAccessTokenSecretName: ""
 
+# URL to a Splunk instance to send data to.
+# If empty no data will be sent to Splunk.
+splunkUrl: ""
+
+# Splunk HTTP Event Collector token
+splunkToken: ""
+
+# Use an existing secret for the Splunk token (if set does not create a secret)
+# See templates/secrets.yaml for the format of the secret to use
+splunkTokenSecretName: ""
+
+# Splunk source: https://docs.splunk.com/Splexicon:Source
+splunkSource: ""
+
+# Splunk source type: https://docs.splunk.com/Splexicon:Sourcetype
+splunkSourceType: ""
+
+# Splunk index, optional name of the Splunk index to store the event in
+splunkIndex: ""
+
+# Skip verifying the certificate of the HTTP Event Collector
+splunkSkipTLSVerify: false
+
+# The hostname of the agent sent to Splunk
+splunkHost: "signalfx-agent"
```

Comment:
SignalFx datapoints and events generated by the agent usually have a `host` dimension that the 
Splunk writer could introspect and use.  Regardless, we probably shouldn't hard code this name in the Helm chart unless _host_ means something different in Splunk.

Reply:
I have to check this. I do believe the host property is the host of the collector, not the host you pick values from. The host associated with the metric is added as dimension.

Checking that assumption in the docs and coming up empty, but will keep looking.

Reply:
Host is the origin of the event, and best passed in with hostIDDims looks like. I am going to use the "host" dimension for each event or data point for now to pass in. Let me know if that works.